runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1186,6 +1209,37 @@ TR_ResolvedJ9JITaaSServerMethod::setAttributes(TR_OpaqueMethodBlock * aMethod, T
    setMandatoryRecognizedMethod(mandatoryRm);
    setRecognizedMethod(rm);
 
+   _useCaching = true; // TODO: figure out how to change this variable without retrieving env variable each time a resolved method is created (i.e. feGenEnv(TR_DisableResolvedMethodsCaching))
```

Comment:
The solution is to define _useCaching as a static. If initializing this static does not work with 
`bool TR_ResolvedJ9JITaaSServerMethod::_useCaching = (feGetEnv("TR_ResolvedJ9JITaaSServerMethodCaching") != NULL);`
Then you could set it to true and overwrite it during options processing when we set options for JITaaS. In the end we may want an option rather than an env var.